sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1095,6 +1096,23 @@ export class FeedResponse<TResource> {
     readonly resources: TResource[];
 }
 
+// @public
+export interface FullTextIndex {
```

Comment:
https://github.com/Azure/azure-sdk-for-js/pull/31710/files#r1840859380 - copying comment over

> FullTextSearchIndex, FullTextSearchPath, FullTextSearchPolicy might be clearer, just fullText doesn't tell me much

Reply:
will update in a separate PR.